plugins/plugin-dotenv/README.md

```diff
@@ -29,4 +29,7 @@ SNOWPACK_PUBLIC_ENABLE_FEATURE=true
 
 #### Plugin Options
 
-None
+| Name       | Type     | Description                                           |
+| :--------- | :------- | :---------------------------------------------------- |
+| `dir`      | `string` | (optional) Where to find `.env` files. Default is `.` |
+| `defaults` | `object` | (optional) Default environment variables.             |
```

Comment:
I'd like to avoid the "defaults" config for now. We just merged the ability to set `config.env` and pass env variables directly through there, and I'd like to let that settle a bit first before we add more features around env variable handling. 

DIr makes a lot of sense though, would love to see that added here.

Reply:
Ok, `config.env` is a better place.